db/config.cc

```diff
@@ -153,6 +166,18 @@ struct convert<db::config::seed_provider_type> {
     }
 };
 
+template<>
+struct convert<db::duration_in_ms_type> {
+    static bool decode(const Node& node, db::duration_in_ms_type& v) {
+        uint32_t tmp;
+        if (!convert<uint32_t>::decode(node, tmp)) {
+            return false;
```

Comment:
It will be nice (one day) do add support for unit suffixes so the user can write 2s or 2000ms and get the same result.

Reply:
I thought about it and if we would have broken backward compatibility I would use duration is type (vs adding a new type).
I decided against it because in yaml, 1000 and 1s are not the same.

But it's definitely an option.